frontend/pages/setting/email.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <Loading :loading="loading" />
+    <v-form ref="EmailForm">
+      <UserFormPassword
+        v-model="user.password"
+        noIcon
+        noValidation
+        label="パスワード"
+      />
+
+      <UserFormEmail v-model="user.email" noIcon label="新しいメールアドレス" />
+
+      <ErrorMessage :message="message" />
+
+      <Btn @click="updatePassword">メールアドレスを変更する</Btn>
```

Comment:
メールアドレスの変更なので関数名はupdateEmailとかではないでしょうか？
```suggestion
      <Btn @click="updateEmail">メールアドレスを変更する</Btn>
```

Reply:
修正しました。